src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/BookingEntity.kt

```diff
@@ -133,7 +133,7 @@ interface BookingRepository : JpaRepository<BookingEntity, UUID> {
   fun findAllByCrn(crn: String): List<BookingEntity>
 
   @Query(
-    "SELECT b From BookingEntity b WHERE b.adhoc IS TRUE or b.adhoc IS NULL",
+    "SELECT b From BookingEntity b WHERE b.application = :application AND b.adhoc IS TRUE or b.adhoc IS NULL",
```

Comment:
Should we have brackets here? i.e:

```suggestion
    "SELECT b From BookingEntity b WHERE b.application = :application AND (b.adhoc IS TRUE or b.adhoc IS NULL)",
```

Reply:
Or maybe `IS NOT FALSE` would cover both bases?

Reply:
great spot, should be fixed now